Javascript application for fuzzing (#2)
CodeIntelligenceTesting/ci-fuzz-cli-tutorials: js_ts/nodejs-js/README.md

> # Nodejs JS demo application

Reviewer: ```suggestion
# NodeJS demo application
```

> ## TODO application
> 
> ### Usage of application

Reviewer: ```suggestion
## Functionality
```

> It is simple nodejs express TODO application which has several functionalities, such as, adding, 

Reviewer: ```suggestion
It is simple nodeJS express TODO application, which has several functionalities, such as, adding, 
```

> deleting, listing TODOs, deleting whole json file and command execution in the server. The application creates 

Reviewer: ```suggestion
deleting single TODOs, and listing TODOs. Additional functionality includes, deleting the whole TODO list json file and command execution on the server. The application creates 
```

> todo.json file in the root folder of the application.

Reviewer: ```suggestion
a `todo.json` file in the root folder of the application as a database to save all added TODOs.
```

> Endpoints: 
>     
> `/api/add?todo=<todo>&deadline=<deadline>`
>     
> `/api/delete?id=<id>`
> 
> `/api/list`
> 
> `/api/deleteList`
> 
> `/api/server?command=<command>`

Reviewer: ```suggestion
Available endpoints: 
    
```
/api/add?todo=<todo>&deadline=<deadline>
/api/delete?id=<id>
/api/list
/api/deleteList
/api/server?command=<command>
```

```
